profiler/src/profile_contraction_scale.cpp

```diff
@@ -99,7 +99,9 @@ int profile_contraction_scale(int argc, char* argv[])
                 assign_default_strides(cde_layout, StridesD, {M[0], M[1], N[0], N[1]});
             }
 
-            bool pass = ck::profiler::profile_contraction_impl<ALayout,
+            constexpr ck::index_t NumDim = 2;
```

Comment:
Isn't it possible to profile also 6D cases ?

Reply:
Done.